io.asgardio.dotnet.oidc.sdk.sample/App.config

```diff
@@ -0,0 +1,85 @@
+        <conversionPattern value="%date %level [%thread] %type.%method - %message%n" />
+      </layout>
+    </appender>
+    <root>
+      <level value="All" />
+      <!-- If the following line is not included the log file 
+      will not be created even if log4net is configured with this file. -->
+      <appender-ref ref="TestAppender" />
+    </root>
+  </log4net>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+  </startup>
+  <appSettings>
+    <add key="ClientId" value="QzVKW6C6TIUHzaef_KtIfNyUm70a" />
+    <add key="ClientSecret" value="TKlmp3RTqSGUu407DW9VrRyxE0Ma" />
+    <add key="AuthorizationEndpoint" value="https://localhost:9443/oauth2/authorize" />
+    <add key="TokenEndpoint" value="https://localhost:9443/oauth2/token" />
+    <add key="UserInfoEndpoint" value="https://localhost:9443/oauth2/userinfo" />
+    <add key="LogoutEndpoint" value="https://localhost:9443/oidc/logout" />
+    <add key="RedirectURI" value="http://localhost:8080/callback/" />
+    <add key="PostLogoutRedirectURI" value="http://localhost:8080/postlogout/" />
+    <add key="ClientSettingsProvider.ServiceUri" value="" />
+  </appSettings>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" />
+  </startup>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.IdentityModel.Tokens.Jwt" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-5.6.0.0" newVersion="5.6.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.IdentityModel.Tokens" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-5.6.0.0" newVersion="5.6.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
+  </runtime>
+  <system.web>
+    <membership defaultProvider="ClientAuthenticationMembershipProvider">
+      <providers>
+        <add name="ClientAuthenticationMembershipProvider" type="System.Web.ClientServices.Providers.ClientFormsAuthenticationMembershipProvider, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" serviceUri="" />
+      </providers>
+    </membership>
+    <roleManager defaultProvider="ClientRoleProvider" enabled="true">
+      <providers>
+        <add name="ClientRoleProvider" type="System.Web.ClientServices.Providers.ClientRoleProvider, System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" serviceUri="" cacheTimeout="86400" />
+      </providers>
+    </roleManager>
+  </system.web>
+</configuration>
```

Comment:
Missing new line at EOF.

Reply:
Resolved in https://github.com/wso2-extensions/identity-sdks-dotnet/pull/6/commits/ad95c5751c2becdbce174d985942960300fed9bc.